libraries/standard/mqtt/integration-test/mqtt_system_test.c

```diff
@@ -496,11 +493,16 @@ static void handleAckEvents( MQTTPacketInfo_t * pPacketInfo,
 
 static void eventCallback( MQTTContext_t * pContext,
                            MQTTPacketInfo_t * pPacketInfo,
-                           uint16_t packetIdentifier,
-                           MQTTPublishInfo_t * pPublishInfo )
+                           MQTTDeserializedInfo_t * pDeserialized )
 {
+    MQTTPublishInfo_t * pPublishInfo;
```

Comment:
Default initialization of stack variables?
```suggestion
    MQTTPublishInfo_t * pPublishInfo = NULL;
```

Reply:
This isn't necessary since the pointer is unconditionally set to `pDeserialized->pPublishInfo` seven lines later, after the asserts. It doesn't hurt to add it though so I made the change